README.md

```diff
@@ -1,5 +1,7 @@
 # Project-Tactics
-An open-source game development framework inspired by Final Fantasy Tactics, letting you create your own unique tactical RPG adventures.
+An open-source game development framework inspired by Final Fantasy Tactics, letting you create your own unique tactical RPG adventures. The project goal is to allow users to create a set of battlegrounds to share with other players.
```

Comment:
Based on the use case I can change this phrase later. You can merge it as is for now.